DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -141,7 +144,7 @@
     private final AtomicBoolean retrievingSpeakers = new AtomicBoolean(false);
 
     private ReadOnlyListWrapper<Track> tracks = new ReadOnlyListWrapper<>(FXCollections.observableArrayList());
-    private ReadOnlyListWrapper<ProposalType> proposalTypes = new ReadOnlyListWrapper<>(FXCollections.observableArrayList());
```

Comment:
Now that `ProposalType` is no longer required. Should we get rid of `ProposalType` and `ProposalTypes` model?

Reply:
We can merge this PR and I can take care of this.

Reply:
Done